config/test_cases/test_pull_request_openshift_ansible_tox.yml

```diff
@@ -18,6 +18,20 @@ actions:
     repository: "openshift-ansible"
     script: |-
       tox
+  - type: "script"
```

Comment:
I propose we start with an opt-in flow. Is there an easy way to parse a comment like [test:integration] or similar to trigger a job? IOW, is the logic that parses [merge] easy to extend?